src/engine/source/indexerconnector/include/indexerConnector/indexerConnector.hpp

```diff
@@ -53,6 +76,7 @@ class EXPORTED IndexerConnector final : public IIndexerConnector
     std::string m_indexName;
     std::mutex m_syncMutex;
     std::unique_ptr<ThreadDispatchQueue> m_dispatcher;
+    IndexerConnectorOptions m_indexerConnectorOptions;
```

Comment:
Why do we store the configuration? Is it necessary to have a state, I don't see it used after initialization.

Reply:
Removed